### ISSUE TITLE
[Deploy] YNAB

### ISSUE THREAD

**Issue by davereid** [meetfranz/plugins#216]
### Service
[You Need a Budget (YNAB)](https://www.ynab.com/) is a multi-platform personal budgeting program based on the envelope method.

### Link to your repository
https://gitlab.com/davereid/franz-recipe-ynab

### Additional Information
Licensed under MIT

**Comment by chisaipete** [comment]
Hah, I should have looked through the issues, I built a nearly identical plugin too:

https://github.com/chisaipete/recipe-franz-ynab

**Comment by davereid** [comment]
@chisaipete Yes they look similar, I'd be open to any improvements. I will note that I added support for indirect notifications when your accounts have items to categorize or import.

**Comment by chisaipete** [comment]
Oh, nice! I'll close mine and use yours!
